Update azure-stack-replace-node.md

Paxilein opened the pull request:
We have a broken storage because of this... So I think it's good to warn future users....

azurecla commented:
Hi __@adfteam__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
@spelluru 
Please review and approve/disapprove this PR here, thanks.

spelluru commented:
I have made exact same changes as part of another PR today. Please close this. I have already communicated this to Wee Hyong. Thanks.

spelluru commented:
Here is the duplicate PR: https://github.com/Azure/azure-content-pr/pull/14065.

Paxilein committed:
Update azure-stack-replace-node.md

We have a broken storage because of this... So I think it's good to warn future users....

mattbriggs reviewed (approved):
#sign-off

GitHubber17 committed:
Update azure-stack-replace-node.md
